packages/chart/src/scss/DataTable.scss

```diff
@@ -1,5 +1,10 @@
 .data-table-container {
   margin: 20px 0 0;
+
+  &.brush-active {
```

Comment:
Would it be better to remove this and add `mt-4` or something similar conditionally if the brush chart is active?

Reply:
Bootsrap does not support margins over 50 pixels.